src/core/model/Stroke.cpp

```diff
@@ -392,14 +417,16 @@ void Stroke::setLastPressure(double pressure) {
         assert(pressure != Point::NO_PRESSURE);
         Point& back = this->points.back();
         back.z = pressure;
-        updateBounds(Element::x, Element::y, Element::width, Element::height, snappedBounds, back, 0.5 * pressure);
```

Comment:
Why is this necessary? The last point's pressure is not used in rendering, unless the stroke has exactly one point, in which case you end up calling `updateBounds` from `updateBoundsLastTwoPressures`